helm-charts/charts/sentinel/values.yaml

```diff
@@ -0,0 +1,72 @@
+# Default values for sentinel.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+livenessPort: 8081
+
+environments:
+ - name: SPIFFE_ENDPOINT_SOCKET
+   value: "unix:///spire-agent-socket/agent.sock"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+ingress:
+  enabled: false
+  className: ""
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    - host: chart-example.local
+      paths:
+        - path: /
+          pathType: ImplementationSpecific
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
```

Comment:
Not sure if we shall allow ingress to sentinel: It is way too risky, as it has a direct and unprotected reach to Aegis Safe.

Though it is false by default; and it might also be useful to test things out maybe. 
 
I am okay since `enabled: false` means this is still secure by default.

Reply:
Agreed, this was added as part of helm charts template. Removed from all 3 sub charts. Thanks :)

Reply:
Done.